### DISCUSSION TITLE
Set the 'strength' increment?

### DISCUSSION THREAD

**Discussion by holdengand** [Acly/krita-ai-diffusion#1179] [Q&A]
I just updated and the strength increment seems to be 10% increments now, pretty sure it was 5 before? Anyway, I don't want to have to manually type the changes every time, I'd like to set it at 2 as 5% was generally to much in my opinion or at least for how I use it anyway. There is no setting to adjust this increment in the built in settings, and I assume I can change this manually in the python files, but I'm not actually sure what exactly I'd need to change.

I thought I'd found it in the control.py file in the UI files, and changed 1 or 2 things and restarted without making any difference. So rather then screw around until I break it I figured I'd ask here.

Where am I meant to find this setting?

**Comment by Acly** [comment]
The minimum should be 10% (slider) / 5% (text box) as before.

What you might be observing is that it snaps to the next meaningful value. This depends on the total number of sampling steps. If you only have 10 steps, anything lower than 10% increment doesn't make sense - it will be rounded anyway.

#838 has a (very lengthy) discussion on this

**Reply by holdengand** [reply]
Thanks  for responding. I see. I kind of discovered the answer myself after writing this after I messed around with some settings and toggled on the "Show Steps" option. I think part of my confusion was that I wasn't regularly playing with LCM or Turbo models last time I was messing with this, so my steps were never low enough to even realize.

I think I have always been misunderstanding what this strength setting was actually doing then all this time.
